src/_static/dashboard-wrapper.html

```diff
@@ -0,0 +1,104 @@
+<link rel="import" href="/static/polymer-1.2.0/polymer/polymer.html">
+<link rel="import" href="/static/polymer-1.2.0/iron-ajax/iron-ajax.html"/>
+<link rel="import" href="/static/polymer-1.2.0/paper-spinner/paper-spinner.html"/>
+<link rel="import" href="/modules/coursebuilder-student-dashboard/_static/student-dashboard.html">
```

Comment:
@andela-iakande Should we have this in /static?

Reply:
@Chege Though these import paths are not needed here they are only required in student-dashboard.html I will remove it. However, the reason for the /static in our imports path is that it contains all static generated files by default during initialization of the project and when we download dependencies. This happens when we run script on start_in_shell.sh or common.sh